libfreerdp/core/connection.c

```diff
@@ -773,7 +773,7 @@ BOOL rdp_client_connect_auto_detect(rdpRdp* rdp, wStream* s)
 		{
 			if (channelId == rdp->mcs->messageChannelId)
 			{
-				UINT16 securityFlags;
+				UINT16 securityFlags = 0;
```

Comment:
Was this a compiler warning? - As the securityFlags are always initialized because they are read below.